### PR TITLE
Parse 0 frequency values as the other non-zero values

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR fixes a bug that causes variant frequencies not to be saved when their value is 0.
I've noticed about it because of https://github.com/Clinical-Genomics/MIP/issues/1420

**How to test**:
1. Done by automatic tests

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN, MM
- [x] tests executed by Travis CI

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready for review!

File: scout/parse/variant/frequency.py [hunk 1]
```diff
@@ -25,37 +25,37 @@ def parse_frequencies(variant, transcripts):
 
     for test_key in thousand_genomes_keys:
         thousand_g = parse_frequency(variant, test_key)
-        if thousand_g:
+        if thousand_g is not None:
```
Comment (dnil) [inline scout/parse/variant/frequency.py]: :+1:
Python is sometimes 😈 ..

File: tests/parse/test_frequency.py [hunk 2]
```diff
@@ -101,6 +101,29 @@ def test_parse_sv_frequencies_clingen(cyvcf2_variant):
...
+    variant = one_vep97_annotated_variant
+
+    # GIVEN a variant with a gnomad frequency set to 0
+    variant.INFO["GNOMADAF"] = "0.000"
+
+    # WHEN frequencies are parsed
+    frequencies = parse_frequencies(variant, [])
+
+    # THEN assert that the right frequency is returned
+    assert frequencies["gnomad"] == float(variant.INFO["GNOMADAF"])
```
Comment (dnil) [inline tests/parse/test_frequency.py]: :+1:

File: scout/parse/variant/frequency.py (hunk 1 above)
Comment (moonso) [reply scout/parse/variant/frequency.py]: The linters like this syntax :) I have actually learned to write better code thanks to wading through linting hell for a while

**Pull request merged**